javascript/pop_up.js

```diff
@@ -0,0 +1,161 @@
+        dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the
+        releaLorem Ipsum is simply dummy text of the printing and typesetting ever since the 1500s, when an unknown printer took
+        a galley of type veris lapoa todoe.</p>
+        <ul class="window-list">
+            <li>
+                <div class="v-line"></div>
+            </li>
+            <li>
+                <h3>html</h3>
+            </li>
+            <li>
+                <div class="v-line"></div>
+            </li>
+            <li>
+                <h3>Ruby on rails</h3>
+            </li>
+            <li>
+                <div class="v-line"></div>
+            </li>
+            <li>
+                <h3>css</h3>
+            </li>
+            <li>
+                <div class="v-line"></div>
+            </li>
+            <li class="wdesk">
+                <h3>Github</h3>
+            </li>
+            <li class="wdesk">
+                <div class="v-line"></div>
+            </li>
+            
+        </ul>
+        <div class="window-buttons">
+            <a href="https://khanetmoi.github.io/MicroversePortfolio/">
+            <button class="Window-project">
+                <h3>See Live</h3><img class="window-power-logo" src="images/Icon.png" alt="">
+            </button> </a>
+           <a href="https://github.com/Khanetmoi/MicroversePortfolio/tree/master"> <button class="window-source">See source <i class="fa-brands fa-github"></i></button> </a>
+        </div>`;
+
+body.appendChild(WpopUp);
+console.log(body);
+
+console.log(WpopUp);
+
+const mappeur = projects.map((fuji) => `<div class=${fuji.side}>
+            <div class="preview"></div>
+            <div class="comments">
+                <h2 class="title">${fuji.projectTitle}</h2>
+                <p>${fuji.projectDescription}</p>
+                <ul> ${fuji.technologies.map((frame) => `<li><h3>${frame}</h3></li>`).join('')}
+                </ul>
+
+                <button class=${fuji.buttonClass}>See Project</button>
+            </div>
+        </div>`).join('');
+
+const content = document.getElementById('bgImg2');
+
+content.innerHTML += mappeur;
+
+const button = document.getElementsByClassName('Pbutton');
+
+console.log(button);
+
+const PopUpWindow = document.getElementById('window');
+const Close = document.getElementById('close');
+
+console.log(PopUpWindow);
+
+button[0].addEventListener('click', () => {
+  PopUpWindow.classList.add('popUpAppear');
+  body.style.overflowY = 'hidden';
+});
+button[1].addEventListener('click', () => {
+  PopUpWindow.classList.add('popUpAppear');
+  body.style.overflowY = 'hidden';
+});
+button[2].addEventListener('click', () => {
+  PopUpWindow.classList.add('popUpAppear');
+  body.style.overflowY = 'hidden';
+});
+button[3].addEventListener('click', () => {
+  PopUpWindow.classList.add('popUpAppear');
+  body.style.overflowY = 'hidden';
+});
```

Comment:
- _(optional)_ You did a great job, but it can be improved. I recommend looping through the array of objects. Adding an event listener to each element in the array manually is not a very good practice.